Rules.md

```diff
@@ -15400,12 +15426,12 @@ extension Foo {
 ```
 
 ```swift
-@objc
```

Comment:
It seems from the diff that this triggering example was removed. Could you add it back to make sure it is triggered?

Reply:
I moved it to be a non-triggering example because the declaration is private (similar to https://github.com/realm/SwiftLint/pull/2501)

Reply:
Ok, i see, i agree, its a good idea to have it in the non triggering too.
However it is redundant to mark any attribute in an already `@objc` marked `extension`, regardless what the `ACL` is.

Reply:
Added it back in 0c92b948a9ebe52e30b44371ebf9cff9c4de8832